scrape/manager.go

```diff
@@ -14,14 +14,19 @@
 package scrape
 
 import (
+	"encoding"
 	"fmt"
+	"hash/fnv"
+	"net"
+	"os"
 	"reflect"
 	"sync"
 	"time"
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 
+	"github.com/prometheus/common/model"
```

Comment:
We're trying to get rid of usage of this library.

Reply:
Any suggestions?

Reply:
should I use `github.com/prometheus/prometheus/pkg/labels` instead?

I can add this to GlobalConfig:
```
// GetExternalLabels wraps external labels as a labels.Labels.
func (c *GlobalConfig) GetExternalLabels() labels.Labels {
	labelSet := make(map[string]string, len(c.ExternalLabels))
	for n, v := range c.ExternalLabels {
		labelSet[string(n)] = string(v)
	}

	return labels.FromMap(labelSet)
}
```

but as this isn't related to the Jitter, let's leave it as is and do clean-up in follow up PRs, I can help you out with it.

Didn't find any issue regarding liquidation the `github.com/prometheus/common/model` library.

<img width="609" alt="image" src="https://user-images.githubusercontent.com/922743/53886929-07c19d00-4019-11e9-9395-783ccda3af43.png">

Reply:
model.Labelset is already a map[string]string, so you can use FromMap directly. You can also re-use its Hash method.

Reply:
the problem is, the `model.Labelset` isn't a `map[string]string`, it's `map[LabelName]LabelValue` even if `LabelName` and `LabelValue` are strings, so it's not the same and golang can't do type conversion.

Reply:
You can cast it

Reply:
what am I doing wrong?
`labelSet := map[string]string(cfg.GlobalConfig.ExternalLabels)`

```
cannot convert cfg.GlobalConfig.ExternalLabels (type model.LabelSet) to type map[string]string
```

https://play.golang.org/p/EwQjDtB00dJ 

```
prog.go:13:23: cannot convert ls2 (type LabelSet2) to type map[string]string
```
where as LabelSet works as mentioned before.

Reply:
Right you are.